data/index.html

```diff
@@ -162,14 +206,31 @@ <h2 class="demoHeaders">Dave's Fire NightDriverStrip</h2>
                 dataType: "text",
                 success: function (responseData, textStatus, jqXHR) {
                     var obj = JSON.parse(responseData);
-                    UpdateEffectList(obj);
+                    UpdateAll(obj);
                 },
                 error: function (responseData, textStatus, errorThrown) {
-                    UpdateEffectList(null);
+                    UpdateAll(null);
                     setTimeout(RequestEffectList, 2000);

```

Comment:
I think this `setTimeout` can be removed. The function will be called again after 2 seconds anyway due to the `setInterval` below.

Reply:
Good catch! Yes you're absolutely right, I'll remove that now!